### PR TITLE
docs(Config Schema): Add docs for defineTopLevelProperty and helpers map

### PR DESCRIPTION
**Pull request by fredericbarthelet**
Closes: #8616

### CODE REVIEW

File: docs/providers/aws/guide/plugins.md [hunk 1]
```diff
@@ -417,15 +417,84 @@ Command names need to be unique. If we load two commands and both want to specif
...
+          customProperty: foobar # <-- use defineFunctionEventProperties
+```
 
 We'll walk though those helpers. You may also want to check out examples from [helpers tests](tests/fixtures/configSchemaExtensions/test-plugin.js)
 
+#### `defineTopLevelProperty` helper
+
+If your plugin requires additional top-level properties (like `provider`, `custom`, `service`...), you can use the `defineTopLevelProperty` helper to add their definition.
+
+Considering the following exemple
```
Comment (pgrzesik) [inline docs/providers/aws/guide/plugins.md:454]: Minor typo: `exemple` -> `example`

File: docs/providers/aws/guide/plugins.md [hunk 2]
```diff
@@ -417,15 +417,84 @@ Command names need to be unique. If we load two commands and both want to specif
...
+```yml
+// serverless.yml
+
+service: my-service
+
+yourPlugin:
+  someProperty: foobar
+```
+
+you'll need to add vaalidation rules as described below:
```
Comment (pgrzesik) [inline docs/providers/aws/guide/plugins.md:465]: Minor typo: `vaalidation` -> `validation`

**Review by pgrzesik — CHANGES_REQUESTED** [review]
Well done :raised_hands: I've only caught two small typos, otherwise it looks great :+1:

**Comment by fredericbarthelet** [comment]
Thanks for your review @pgrzesik :)
I updated this PR fixing the typos ;)

**Review by pgrzesik — APPROVED** [review]
:tada: Thank you @fredericbarthelet

**Pull request merged**